code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -7,13 +7,19 @@
 	slot_flags = SLOT_BELT
 	attack_verb = list("whipped", "lashed", "disciplined")
 	max_integrity = 300
+	var/content_overlays = FALSE //If this is true, the belt will gain overlays based on what it's holding
 
 /obj/item/weapon/storage/belt/update_icon()
 	cut_overlays()
-	for(var/obj/item/I in contents)
-		add_overlay("[I.name]")
+	if(content_overlays)
+		for(var/obj/item/I in contents)
+			add_overlay(icon('icons/obj/clothing/belt_overlays.dmi', I.icon_state))

```

Comment:
no.
this should be a ```mutable_appearance```.

What even went through your head to make you think ```icon()``` was acceptable here?